benchmarks/nemo/tracer_advection/scripts/omp_gpu_trans.py

```diff
@@ -36,8 +36,7 @@
 ''' PSyclone transformation script to insert OpenMP Target Loop directives
 to the outermost loop that is parallelisable, including implicit loops. '''
 
-from psyclone.psyir.transformations import OMPTargetTrans
-from psyclone.transformations import OMPLoopTrans
+from psyclone.psyir.transformations import OMPTargetTrans, OMPLoopTrans
 from utils import insert_explicit_loop_parallelism, normalise_loops
 
 
```

Comment:
pylint complains a lot about the indentation in this file!

Reply:
I fixed them now.

Reply:
Weird. I've just tried again and it gets 10/10 now :-)